### PR TITLE
Added more points to what's next section under deployment page

### LINKED ISSUE #38159: Update What's next section under Deployment page
**This is a Feature Request**
**What would you like to be added**
I would like to add below references under `What's next section` of `Deployment` page.
1. Using kubectl to [Create a Deployment](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/)
2. Interactive Tutorial - [Deploying an App](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive/)

**Why is this needed**
Above Reference links providing interactive tutorial sections for Deployment which are more helpful to the user to understand the concept clearly and provide hands-on experience.  
**Comments**
Page to update: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

### PR DESCRIPTION
**Pull request by T-Lakshmi**
This PR updated what's next section under deployment page by adding 2 necessary points.
Fixes #38159.

### CODE REVIEW

**Comment by T-Lakshmi** [comment]
Hi mehabhalodiya, shannonxtreme,
   PTAL!

**Comment by mehabhalodiya** [comment]
Hey @T-Lakshmi 
I would highly recommend you to wait for the `triage-accepted` label on your raised issue, then one of the reviewers can look ahead on this. Thanks!

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -1222,5 +1222,7 @@ it is created.
 * `Deployment` is a top-level resource in the Kubernetes REST API.
   Read the {{< api-reference page="workload-resources/deployment-v1" >}}
   object definition to understand the API for deployments.
-* Read about [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how
+* [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/workloads/controllers/deployment.md:1225]: Please retain the original sentence.

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 2]
```diff
@@ -1222,5 +1222,7 @@ it is created.
...
   you can use it to manage application availability during disruptions.
+* Using kubectl to [Create a Deployment](/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/).
+* Interactive Tutorial - [Deploying an App](/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive/).
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/workloads/controllers/deployment.md:1228]: Per [comment](https://github.com/kubernetes/website/issues/38159#issuecomment-1495505827), this is no longer required. Please remove this.

Comment (divya-mohan0209) [reply content/en/docs/concepts/workloads/controllers/deployment.md:1228]:
```suggestion
```

**Comment by T-Lakshmi** [comment]
Hi @divya-mohan0209 ,
Thanks for spending your time to review this PR. I have updated the suggested changes by you.
could you please take an another look.

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 3]
```diff
@@ -1231,5 +1231,5 @@ it is created.
 * `Deployment` is a top-level resource in the Kubernetes REST API.
   Read the {{< api-reference page="workload-resources/deployment-v1" >}}
   object definition to understand the API for deployments.
-* Read about [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how
-  you can use it to manage application availability during disruptions.
+* [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how you can use it to manage application availability during disruptions.
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/workloads/controllers/deployment.md:1234]:
```suggestion
* Read about [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how you can use it to manage application availability during disruptions.
```

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 4]
```diff
@@ -1231,5 +1231,5 @@ it is created.
...
+* Using kubectl to [Create a Deployment](/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/).
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/workloads/controllers/deployment.md:1235]:
```suggestion
* Use kubectl to [Create a Deployment](/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/).
```

Comment (T-Lakshmi) [reply content/en/docs/concepts/workloads/controllers/deployment.md]: Hi @divya-mohan0209 ,
Incorporated the suggested changes, PTAL!

**Comment by T-Lakshmi** [comment]
Hi,
Can anyone please take a look into this PR.

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Comment by T-Lakshmi** [comment]
@divya-mohan0209,
Thanks for providing lgtm.
Can anyone please approve PR?

File: content/en/docs/concepts/workloads/controllers/deployment.md [hunk 5]
```diff
@@ -1231,5 +1231,5 @@ it is created.
 * `Deployment` is a top-level resource in the Kubernetes REST API.
   Read the {{< api-reference page="workload-resources/deployment-v1" >}}
   object definition to understand the API for deployments.
-* Read about [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how
-  you can use it to manage application availability during disruptions.
+* Read about [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) and how you can use it to manage application availability during disruptions.
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/controllers/deployment.md:1234]: Why don't you wrap the long line as we do for the bullet on line 1231?

Comment (T-Lakshmi) [reply content/en/docs/concepts/workloads/controllers/deployment.md:1240]: Hi @tengqm,
Applied your suggestions, could you please take a look into it once!!

Comment (shannonxtreme) [reply content/en/docs/concepts/workloads/controllers/deployment.md:1240]: It's still not wrapped, try to line break at ~80 characters similar to the previous version.

Comment (T-Lakshmi) [reply content/en/docs/concepts/workloads/controllers/deployment.md]: @shannonxtreme, @tengqm,
Hope i fixed the issue now, could you PTAL!!

Deploy preview: https://deploy-preview-38230--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/controllers/deployment/#what-s-next

**Pull request merged**